google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/sync/datacatalog_synchronizer.py

```diff
@@ -166,7 +216,7 @@ def __create_entry_factory(self):
     def __create_tag_factory(self):
         return self._get_tag_factory()(self.__metadata_definition)
 
-    def __create_tag_templates(self):
+    def __create_tag_templates(self, metadata):
```

Comment:
```suggestion
    def __create_tag_templates(self, sql_objects_metadata):
```
Makes sense?

Reply:
This contains both the `base_metadata` and `sql_objects_metadata` so I'd like to keep just metadata.